02 Writing Algorithms/18 Consolidating Data/01 Key Concepts/05 Update Aggregated Bars.html

```diff
@@ -10,9 +10,8 @@ <h4>Manual Updates</h4>
     self.consolidator.Update(trade_bar)
 
 # Example 2: Update the consolidator with data from a history request
-history = self.History(self.symbol, 30, Resolution.Minute)
-for index, row in history.iterrows():
-    trade_bar = TradeBar(index[1], self.symbol, row.open, row.high, row.low, row.close, row.volume)
```

Comment:
I think we should keep this example because if the user has a dataframe, he might want to reuse it.
```python
# Example 2: Update the consolidator with data from a history request
# 2.1: Using the list of TradeBar return type
...
# 2.2: Using the pandas.Dataframe return type
...
```